test/CSharp/CodeCracker.Test/Refactoring/IntroduceFieldFromConstructorTest.cs

```diff
@@ -128,11 +128,12 @@ namespace ConsoleApplication1
     {
         class TypeName
         {
+            private readonly int par1;
             private readonly int par;
 
             public TypeName(int par)
             {
-               this.par = par;
+               this.par1 = par;
```

Comment:
@carloscds This is not supposed to happen. The argument `par` and the field `par` have the same and can be merged together. This test `FieldAlreadyExistsAndMatchesType` verified that.
The bug only happens when you have an event or some other members, like that:

````csharp
class Foo
{
  public event Action a;
  Foo(int a)
  {
  }
}
````